breeze

```diff
@@ -904,7 +904,7 @@ if [[ ${INITIALIZE_LOCAL_VIRTUALENV} == "true" ]]; then
         if [[ ${SYSTEM} == "Darwin" ]]; then
             echo "  brew install sqlite mysql postgresql"
         else
```

Comment:
There are many more Linuxes than Debian ;)

Reply:
Yep. but this is Darwin :) =-> MacOS